teku/src/main/java/tech/pegasys/teku/cli/deposit/EncryptedKeystoreWriter.java

```diff
@@ -39,16 +40,19 @@
   private final String validatorKeyPassword;
   private final String withdrawalKeyPassword;
   private final Path outputPath;
+  private final Consumer<String> log;
 
   public EncryptedKeystoreWriter(
       final SecureRandom secureRandom,
       final String validatorKeyPassword,
       final String withdrawalKeyPassword,
-      final Path outputPath) {
+      final Path outputPath,
+      final Consumer<String> log) {
```

Comment:
commandOuput instead of log? This has different purpose to the main logger

Reply:
updated variable.